components/dashboard/src/settings/Integrations.tsx

```diff
@@ -613,7 +613,7 @@ function GitIntegrationModal(props: ({
             )}
         </div>
         <div className="flex justify-end mt-6">
-            <button className="disabled:opacity-50" onClick={() => validate() && activate()} disabled={!!validationError || busy}>Activate Integration</button>
+            <button onClick={() => validate() && activate()} disabled={!!validationError || busy}>Activate Integration</button>
```

Comment:
**issue**: Could we also remove the hover state when this is disabled? Same for updating git permissions.